### PR TITLE
Document new private location Prometheus metrics

### PR DESCRIPTION
**Pull request by clample**
I recently added new private location metrics to Prometheus. This PR adds them to our documentation as well. It might make sense to add more thorough documentation in the private location section of the docs, but for now this PR adds some basic info to the Prometheus section.

### CODE REVIEW

**Comment by clample** [comment]
Current dependencies on/for this PR:
* main
  * **PR #431** <a href="https://app.graphite.dev/github/pr/checkly/checklyhq.com/431" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/checkly/checklyhq.com/431?utm_source=stack-comment).

**Comment by umutuzgur** [comment]
I leave this to Alex

File: site/content/docs/integrations/prometheus.md [hunk 1]
```diff
@@ -55,6 +78,7 @@ Notice that:
 - The `checkly_check_status` metric has `muted` and `activated` labels, reflecting if a check is sending out alerts or is actually
 running.
 - The `checkly_check_result` metric has a `region` label.
+- The private location "Internal CI" has one Checkly Agent connected. From `checkly_private_location_queue_size` and `checkly_private_location_oldest_scheduled_check_run`, we see that there's no backlog of check run jobs.
```
Comment (checklyalex) [inline site/content/docs/integrations/prometheus.md:84]: Can we link to the https://app.checklyhq.com/settings/account/integrations page here?
When we tell a user to go somewhere on the WebApp, I'd like to link it. Other than that LGTM.

**Review by checklyalex — APPROVED** [review]
LGTM, small tweak https://github.com/checkly/checklyhq.com/pull/431/files#r887099687 
Let's submit and visit later

File: site/content/docs/integrations/prometheus.md (hunk 1 above)
Comment (clample) [reply site/content/docs/integrations/prometheus.md:84]: Good idea :) I'll open another PR

Comment (clample) [reply site/content/docs/integrations/prometheus.md:84]: Done - https://github.com/checkly/checklyhq.com/pull/433

**Pull request merged**